tests/tck/features/utils.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Value parsing
+
+    Scenario: Parsing from text
+    Given A set of string
+        | string                                                | typename  |
+        | 123                                                   | int       |
+        | 3.14                                                  | float     |
+        | true                                                  | bool      |
+        | false                                                 | bool      |
+        | 'string'                                              | str       |
+        | "string"                                              | str       |
+        | "string'substr'"                                      | str       |
+        | 'string"substr"'                                      | str       |
+        | []                                                    | list      |
+        | [1,2,3]                                               | list      |
+        | [-[:e1{}]-,<-[:e2{}]-,-[:e3{}]->]                     | list      |
+        | {1,2,3}                                               | set       |
+        | {}                                                    | dict      |
+        | {k1: 1, k2:true}                                      | dict      |
+        | ('vid':t{p1:0,p2:' '})                                | Node      |
+        | -[:e{p1:0,p2:true}]-                                  | Edge      |
```

Comment:
support edge rank, maybe the format is like:

```text
-[:e@0{p1: 0, p2: ''}]->
```

Reply:
Forgot that.
Addressed.